rgw/v2/tests/s3_swift/test_Mbuckets_with_Nobjects.py

```diff
@@ -239,6 +240,15 @@ def test_exec(config, ssh_con):
                     if config.user_type == "tenanted"
                     else bucket.name
                 )
+                if config.retain_bucket_pol:
+                    log.info(
+                        "Test bucket policy retained at archive site after writing IOs bug-1937618"
+                    )
+                    reusable.retain_bucket_policy(
+                        rgw_conn2, bucket_name_to_create, config
+                    )
+
+                bucket_name_to_create
```

Comment:
Looks like this is an unwanted line..please remove it.

Reply:
done.